presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -950,6 +950,7 @@ public Block getBlock()
         private final BlockConverter elementConverter;
 
         private BlockBuilder builder;
+        private int startingPosition;
```

Comment:
Could we do it in ParquetListConverter, instead of ParquetListEntryConverter? Seems for Struct and Map, the null appending are done there. What do u think?